addons/rose/package.json

```diff
@@ -45,6 +45,7 @@
     "ember-focus-trap": "^1.0.1",
     "ember-named-blocks-polyfill": "^0.2.5",
     "ember-radio-button": "github:yapplabs/ember-radio-button.git#b2ab65f",
+    "ember-test-selectors": "^6.0.0",
```

Comment:
I do not see usage of `ember-test-selectors` within the PR, Why we are adding it as dependency?